resources/views/vendor/filament-panels/components/dismiss-branding-bar.blade.php

```diff
@@ -0,0 +1,53 @@
+{{--
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+--}}
+            
+    @php
+    use App\Settings\CollegeBrandingSettings;
+    $collegeBrandingSettings = app(CollegeBrandingSettings::class);
+    @endphp
+            <div  
+                x-data="{ isVisible: @entangle('isVisible') }" x-show="isVisible"
+                style="--c-600: {{ \Filament\Support\Colors\Color::all()[$collegeBrandingSettings->color][600] }}"
+                class="sticky top-16 z-10 bg-custom-600 text-sm font-medium text-white px-6 py-2 flex items-center h-10"
+            >
+                {{ $collegeBrandingSettings->college_text }}
+                @if ($collegeBrandingSettings->dismissible)
+                <button @click="isVisible = false; $wire.dismiss()" class="hover:text-gray-400 ml-auto">
```

Comment:
I may be wrong, but I think you can get rid of `isVisible = false;` here because you entangled `isVisible` to the PHP property.

Reply:
Done